package.json

```diff
@@ -1,6 +1,10 @@
 {
   "name": "telegraf",
+<<<<<<< HEAD
   "version": "3.39.0",
+=======
+  "version": "5.0.0",
+>>>>>>> develop/develop
```

Comment:
Resolve the conflict

Reply:
I think, you can move to contributing for grammyjs. We deprecate v4, and publish this version as v5 after fix all bugs on this version.

Reply:
Why not release this as v3.40.0, and address design flaws afterwards, in v5.0.0?

@MKRhere might be interested in maintaining v4.

Reply:
Indeed